lib/rubocop/cop/netlify/invalid_model_assignment.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Netlify
+      # This cop checks attribute assignment of Mongoid models
+      #
+      # @example
+      #   # bad
+      #   form.attributes[:email] = "bettse@netlify.com"
+      #
+      #   # good
+      #   form.email = "bettse@netlify.com"
+      class InvalidModelAssignment < Cop
+        MSG = "Assigning to `attributes` will not update record"
+        OBSERVED_METHOD = :attributes
+
+        def on_def(node)
+          return if node.method_name != OBSERVED_METHOD
+
+          node.arguments.each do |argument|
+            if keyword_argument?(argument)
```

Comment:
maybe copy/pasta error? this method is not defined

Reply:
ty, yes.  It was one of the other cops I copied from, but totally unrelated to this.  I got so caught up in getting the pattern :)